mkdocs.yml

```diff
@@ -204,6 +204,7 @@ plugins:
   - search:
       separator: '[\s\-,:!?=\[\]()<>{}"/\\]+|\.(?!\d)|&[lg]t;'
   - awesome-pages
+  - privacy
 
```

Comment:
@rarkins I don't know if merging this PR, with the `privacy` plugin breaks your Google search console stuff? Here's a related PR:

- https://github.com/renovatebot/renovatebot.github.io/issues/216

Reply:
Hopefully not